### PR TITLE
81 telemetry implement onboard telemetry data 2

### PR DESCRIPTION
**Pull request by hanguyen04**

### CODE REVIEW

File: odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    return[fetch_cpu_temperature(), fetch_cpu_usage(), fetch_broker_cpu_usage(), fetch_available_memory()]
+    
+
+
+# CPU Temp
+def fetch_cpu_temperature():
+    temps = psutil.sensors_temperatures(fahrenheit=False)
+    
+    if not temps:
+        sys.exit("can't read any temperature")
```
Comment (jr1221) [inline odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py:16]: can you make this just print the error and return a None list like the other functions?

File: odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return[("TPU/OnBoard/MemAvailable", [mem_available], "MB")]
+    except Exception as e:
+        print(f"Error fetching available memory: {e}")
+        return None
+    
+def main():
+    print(fetch_data())
+
+if __name__ == "__main__":
+    main()
```
Comment (jr1221) [inline odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py:63]: please add a blank line at the end of your file it is good git practice but doesnt change much.

**Review by jr1221 — CHANGES_REQUESTED** [review]
please see two pieces of feedback should be quick.

File: odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py (hunk 1 above)
Comment (hanguyen04) [reply odysseus/odysseus_tree/sources/tpu_telemetry/poll_data/on_board.py]: just fixed them

**Review by jr1221 — APPROVED** [review]
lgtm.  will test with #67

**Pull request merged**